app/ui-react/packages/ui/src/Integration/IntegrationDetailBreadcrumb.tsx

```diff
@@ -0,0 +1,78 @@
+import * as H from 'history';
+import { DropdownKebab } from 'patternfly-react';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { Breadcrumb, ButtonLink } from '../Layout';
+import { IMenuActions } from '../Shared';
+
+export interface IIntegrationDetailBreadcrumbProps {
+  editHref?: H.LocationDescriptor;
+  editLabel?: string | JSX.Element;
+  exportAction?: (e: React.MouseEvent<any>) => any;
+  exportHref?: H.LocationDescriptor;
+  exportLabel?: string | JSX.Element;
+  homeHref?: H.LocationDescriptor;
+  i18nHome?: string;
+  i18nIntegrations?: string;
+  i18nPageTitle?: string;
+  integrationId?: string;
+  integrationsHref?: H.LocationDescriptor;
+  menuActions?: IMenuActions[];
+}
+
+export class IntegrationDetailBreadcrumb extends React.Component<
+  IIntegrationDetailBreadcrumbProps
+> {
+  public render() {
+    return (
+      <Breadcrumb>
+        <Link to={this.props.homeHref!}>{this.props.i18nHome}</Link>
+        <Link to={this.props.integrationsHref!}>
+          {this.props.i18nIntegrations}
+        </Link>
+        <span>{this.props.i18nPageTitle}</span>
+        <div className={'pull-right'}>
```

Comment:
This can't stay here, the `Breadcumb` component creates and `ul` and you can't have a `div` inside it.

You might want to try PF4 [`Level`](http://patternfly-react.surge.sh/patternfly-4/layouts/level/) component to align the `Breadcrumb` and the buttons.

Reply:
Alright, let me know if the changes I made are ok, including the class name.